### PR TITLE
Webpack config: adding oft files to loader test

### PR DESCRIPTION
**Pull request by davidglbr**
Extending loader test to support OFT fonts

### CODE REVIEW

**Comment by coryhouse** [comment]
While I'm fine with the move to support OFT, I'd rather not change the rest of the loader config. Can the rest of the loader config stay the same?

**Comment by davidglbr** [comment]
Hey Cory,

Thank you for this AWESOME repo!
To your question, of course. Only changed the text regex string to be
either TTF or OFT, all the rest is intact.

Cheers,
David
ᐧ

File: webpack.config.prod.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default {
       {test: /\.jsx?$/, exclude: /node_modules/, loader: 'babel'},
       {test: /\.eot(\?v=\d+.\d+.\d+)?$/, loader: 'url?name=[name].[ext]'},
       {test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url?limit=10000&mimetype=application/font-woff&name=[name].[ext]'},
-      {test: /\.ttf(\?v=\d+.\d+.\d+)?$/, loader: 'url?limit=10000&mimetype=application/octet-stream&name=[name].[ext]'},
+      {test: /\.[ot]tf(\?v=\d+.\d+.\d+)?$/, loader: 'file-loader?limit=10000&mimetype=application/octet-stream'},
```
Comment (coryhouse) [inline webpack.config.prod.js]: Can we eliminate the changes on the right?

**Comment by davidglbr** [comment]
My bad,
Yes. Let's please eliminate the change on the right.
Should I update the pull request?

Cheers,
David
ᐧ

**Comment by coryhouse** [comment]
That'd be great, thanks!

**Comment by davidglbr** [comment]
@coryhouse  done

**Comment by coryhouse** [comment]
Thanks! 👍

**Pull request merged**